packages/ui/core-components/src/lib/atoms/inputs/button-group/ButtonGroup.svelte

```diff
@@ -49,29 +56,36 @@
 </script>
 
 <HiddenInPrint enabled={hideDuringPrint}>
-	<div class="inline-flex w-fit max-w-full flex-col mt-2 mb-4 ml-0 mr-2">
+	<div
+		class={display === 'tabs' ? '' : 'inline-flex w-fit max-w-full flex-col mt-2 mb-4 ml-0 mr-2'}
+	>
```

Comment:
Having all of these ternaries that depend on display here is a smell to me, do we need to have other display elements or similar?

Reply:
Sorry i don't follow, but i agree that this may be a bit smelly